plugins/backstage-plugin-flux/src/components/helpers.tsx

```diff
@@ -187,12 +188,15 @@ export function GroupAction({
   resource: Source | Deployment | ImagePolicy;
 }) {
   const { sync, isSyncing } = useSyncResource(resource);
+  const { data } = useGetUserInfo();
+  const user = data?.result?.profile.email || data?.result?.userId;
```

Comment:
There is (_very low_) danger of this not being loaded before the user clicks on suspend.

Perhaps we add `isUserInfoLoading` into the composite `isLoading` below, or just move the `getUserInfo` into the useToggleSuspendResource` hook.

Reply:
Good point, will move it to `useToggleSuspensResource`